buy-resell/products/3rd-party/datev-meinfiskal.md

```diff
@@ -1,76 +1,97 @@
 ---
 slug: /posdealers/buy-resell/products/3rd-party/datev-meinfiskal
 title: DATEV MeinFiskal
----
+---fiskaltrust.Portal

```

Comment:
This part broke the table that we had. Is this intentional?
![image](https://user-images.githubusercontent.com/67687528/187867864-52bb831d-8157-4047-9f73-42a3d593161c.png)